vignettes/articles/oauth.Rmd

```diff
@@ -197,13 +235,31 @@ When you run this code, you'll see something like this, but it will obviously co
 #>  $ location     : chr "Houston, TX"
 ```
 
-### Token management
+### Caching
```

Comment:
In the intro to OAuth session you had written "refresh token: it's okay to store this on your computer" but here you mention the access and refresh tokens will be saved to disk so I'd recommend updating the intro to OAuth session to minimize confusion.

Reply:
especially as you write "you need to think through the consequences of saving the refresh token on disk."